app/models/solidus_stripe/gateway.rb

```diff
@@ -43,11 +34,19 @@ def authorize(amount_in_cents, source, options = {})
     # @option options [Spree::Payment] :originator the payment on which the #capture is being performed (required)
     #
     # @return ActiveMerchant::Billing::Response
-    def capture(_amount_in_cents, _transaction_id, options = {})
-      payment = options[:originator] or raise ArgumentError, "please provide a payment with the :originator option"
-      payment_intent_id = payment.source.stripe_payment_intent_id
+    def capture(amount_in_cents, payment_intent_id, options = {})
+      payment = options[:originator] or
```

Comment:
What's the point of having `payment_intent_id` as an argument if we already have access to it through `options[:originator].payment_intent_id`?

Reply:
But we could ignore the argument, just as before (`_payment_intent_id`). It'd make the method easier to understand, as you wouldn't need to reach the conclusion we're discussing now. But that's a non-blocking from my side.